### PR TITLE
Fixes: Update folders if users reselect the folders after submitting

### LINKED ISSUE #228: Manifests submitted to same folder when app not refreshed between submissions
HTAN Vanderbilt contributor submitted an updated biospecimen manifest, followed by 7 clinical (demographics, diagnosis, etc.) manifests on Nov. 4. She re-selected the appropriate folder and template options from the dropdown menus for each submission, without refreshing the app in between. All manifests were stored in the 'biospecimen' folder as new versions.

Initial biospecimen submission: https://www.synapse.org/#!Synapse:syn25010909.10
Subsequent clinical submissions stored as versions 11-17 of the same Synapse file

### PR DESCRIPTION
**Pull request by rrchai**
- fixes #228 
- fixes the file input is not reset after a successfully submission
- add redirection link to submitted manifest in pop up box

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -170,21 +170,23 @@ shinyServer(function(input, output, session) {
...
   ######## Template Google Sheet Link ########
   observeEvent(c(input$dropdown_folder, input$tabs), {
-    req(input$tabs == "tab_template")
+    req(input$tabs %in% c("tab_template", "tab_upload"))
     tmp_folder_synID <- datatype_list$folders[[input$dropdown_folder]]
     req(tmp_folder_synID != folder_synID()) # if folder changes
 
-    dcWaiter("show", msg = paste0("Getting files in ", input$dropdown_folder, "..."))
     # update selected folder ID
     folder_synID(tmp_folder_synID)
```
Comment (rrchai) [inline server.R:178]: the file Id was not updated in the validation/submission tab

**Comment by rrchai** [comment]
@milen-sage @clarisse-lau 

Current [develop instance](https://shinypro.synapse.org/users/rchai/dc/) is on this branch. Please feel free to test the changes, and let me know if you still have the same issue.

**Comment by milen-sage** [comment]
Thanks for the fast resolution @rrchai ! Feel free to merge when ready.

**Review by clarisse-lau — APPROVED** [review]
thanks @rrchai! 
I submitted a couple manifests making dropdown selections from the 4th tab (Submit & Validate Metadata) without refreshing, and they were stored in the correct folders

**Pull request merged**